plugins/recruit-resources/src/components/CreateVacancy.svelte

```diff
@@ -117,20 +167,6 @@
       showNavigate={false}
       create={{ component: contact.component.CreateOrganization, label: contact.string.CreateOrganization }}
     />
-  </svelte:fragment>
-
-  <AttachmentStyledBox
-    bind:this={descriptionBox}
-    {objectId}
-    _class={recruit.class.Vacancy}
-    space={objectId}
-    alwaysEdit
-    showButtons={false}
-    maxHeight={'card'}
-    bind:content={fullDescription}
-    placeholder={recruit.string.FullDescription}
-  />
-  <svelte:fragment slot="pool">
     <Component
```

Comment:
I think we need to move vacancy template to header, same way we have for Tracker issues.

Reply:
I moved it to the 'Create Vacancy' dialog header like in the tracker:

<img width="724" alt="image" src="https://user-images.githubusercontent.com/84380737/200513095-baf11742-e90b-48a3-a460-19c567f21cfc.png">
<img width="724" alt="image" src="https://user-images.githubusercontent.com/84380737/200513473-75c9d096-aa57-414a-8b65-cf129e55c1ab.png">


Did you mean something else?

Reply:
Nope, this is fine.